resources/sdk/clisdkclient/extensions/config/config.go

```diff
@@ -1,494 +1,498 @@
-        // proxy
-        protocol := viper.Get(fmt.Sprintf("%s.proxy_protocol", profileName))
-        if protocol != nil {
-                proxyconf := ProxyConfiguration{}
-                proxyconf.Port = viper.GetString(fmt.Sprintf("%s.proxy_port", profileName))
-                proxyconf.Protocol = viper.GetString(fmt.Sprintf("%s.proxy_protocol", profileName))
-                proxyconf.Host = viper.GetString(fmt.Sprintf("%s.proxy_host", profileName))
-                userName := viper.Get(fmt.Sprintf("%s.proxy_username", profileName))
-                if userName != nil {
-                        proxyconf.UserName = viper.GetString(fmt.Sprintf("%s.proxy_username", profileName))
-                        proxyconf.Password = viper.GetString(fmt.Sprintf("%s.proxy_password", profileName))
-                }
-                return &proxyconf
-        } else {
-                return nil
-        }
+	// proxy
+	protocol := viper.Get(fmt.Sprintf("%s.proxy_protocol", profileName))
+	if protocol != nil {
+		proxyconf := ProxyConfiguration{}
+		proxyconf.Port = viper.GetString(fmt.Sprintf("%s.proxy_port", profileName))
+		proxyconf.Protocol = viper.GetString(fmt.Sprintf("%s.proxy_protocol", profileName))
+		proxyconf.Host = viper.GetString(fmt.Sprintf("%s.proxy_host", profileName))
+		userName := viper.Get(fmt.Sprintf("%s.proxy_username", profileName))
+		if userName != nil {
+			proxyconf.UserName = viper.GetString(fmt.Sprintf("%s.proxy_username", profileName))
+			proxyconf.Password = viper.GetString(fmt.Sprintf("%s.proxy_password", profileName))
+		}
+		return &proxyconf
+	} else {
+		return nil
+	}
 }
 
 func (c *configuration) String() string {
-        return fmt.Sprintf(`{"profileName": "%s", "environment": "%s", "logFilePath": "%s", "loggingEnabled": "%v", "clientName": "%s", "clientSecret": "%s", "secureLoginEnabled": "%v", "redirectURI": "%s", "accessToken": "%s", "autoPaginationEnabled": "%v","proxyConfiguration": "%s"}`, c.ProfileName(), c.Environment(), c.LogFilePath(), c.LoggingEnabled(), c.ClientID(), c.ClientSecret(), c.SecureLoginEnabled(), c.RedirectURI(), c.AccessToken(), c.AutoPaginationEnabled(), getProxyConfig(c.ProfileName()).String())
+	return fmt.Sprintf(`{"profileName": "%s", "environment": "%s", "logFilePath": "%s", "loggingEnabled": "%v", "clientName": "%s", "clientSecret": "%s", "secureLoginEnabled": "%v", "redirectURI": "%s", "accessToken": "%s", "autoPaginationEnabled": "%v","proxyConfiguration": "%s"}`, c.ProfileName(), c.Environment(), c.LogFilePath(), c.LoggingEnabled(), c.ClientID(), c.ClientSecret(), c.SecureLoginEnabled(), c.RedirectURI(), c.AccessToken(), c.AutoPaginationEnabled(), getProxyConfig(c.ProfileName()).String())
 }
 
 func (a *ProxyConfiguration) String() string {
-        return fmt.Sprintf(`{protocol: %s, host: %s,  port: %s, userName: %v, password: %s}`, a.Protocol, a.Port, a.Host, a.UserName, a.Password)
+	if a == nil {
+		return ""
+	}
```

Comment:
This check is the only thing i added, I don't know why everything is showing up as changed

Reply:
your formatter converted all the spaces to tabs :)